release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1436,6 +1443,11 @@ In the following tables, features are marked with the following statuses:
 |Technology Preview
 |Technology Preview
 
+|Define a vSphere failure domain for a control plane machine set
```

Comment:
Only a nit, but: Noticed the entry before this one used an "-ing" verb ("Managing..."), so maybe switch to "Defining" here (also matches the earlier section heading).

Looking around at the other tables, the "Installation Technology Preview tracker" seems to also favor "-ing", FWIW.

Reply:
Thanks. Good point as consistency is key. Updated.